src/RefractAPI.cc

```diff
@@ -147,7 +147,7 @@ namespace drafter {
         // Add sample value
         if (!parameter.node->exampleValue.empty()) {
             refract::ArrayElement* samples = new refract::ArrayElement;
-            samples->push_back(CreateArrayElement(LiteralToRefract<T>(MAKE_NODE_INFO(parameter, exampleValue), context), true));
+            samples->push_back(LiteralToRefract<T>(MAKE_NODE_INFO(parameter, exampleValue), context));
```

Comment:
Since this code is run in `enum`, the samples should always be an array of `content`. Thus, this change is wrong.